File: Modul 1/Soal 4/PRAK104-2310817210019-NaufalElyzar.c
#include <stdio.h>

int main() {
    int HargaA = 400000;
    int HargaB = 350000;

    int DiskonA = 13;
    int DiskonB = 21;

    int TotalA = HargaA - (HargaA * DiskonA / 100);
    int TotalB = HargaB - (HargaB * DiskonB / 100);

    printf("Harga sepatu A adalah %d\n", HargaA);
    printf("Harga sepatu B adalah %d\n", HargaB);
    printf("Sepatu A mendapat diskon %d%% sehingga harganya menjadi %d\n", DiskonA, TotalA);
    printf("Sepatu B mendapat diskon %d%% sehingga harganya menjadi %d\n", DiskonB, TotalB);

    return 0;
}

File: Modul 1/Soal 4/PRAK104-2310817210019-NaufalElyzar.py
HargaA = 400000
HargaB = 350000

DiskonA = 0.13  
DiskonB = 0.21  

TotalA = HargaA - (HargaA * DiskonA)
TotalB = HargaB - (HargaB * DiskonB)

print(f"Harga sepatu A adalah {HargaA}")
print(f"Harga sepatu B adalah {HargaB}")
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {TotalA:.0f}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {TotalB:.0f}")
